config/site_settings.yml

```diff
@@ -497,7 +497,7 @@ login:
   log_out_strict: false
   pending_users_reminder_delay:
```

Comment:
Instead of altering the values of the existing site settings, I'll prefer if we add a new site setting called `pending_users_reminder_delay_minutes` and then migrate the old site setting over. I think this site setting was added a long time ago but most of our time based site settings specify the units of duration in the name itself. It provides a much better experience for developers using the site settings since they don't have to guess or lookup what the units of the site setting are.